app/src/ui/changes/changed-file.tsx

```diff
@@ -105,6 +107,19 @@ export class ChangedFile extends React.Component<IChangedFileProps, void> {
       })
     }
 
+    if (this.props.status !== FileStatus.Deleted) {
+      items.push(
+        { type: 'separator' },
+        {
+          label: 'Open in external editor',
```

Comment:
This needs to be platform-arized to `Open in External Editor` for `__DARWIN__`.

I'm also not sold on `external editor` here for Windows, but that's because we didn't get around to implementing this in Classic Window - oops. @niik feels on what we should call this?